### ISSUE TITLE
integrate an API so that the music plays itself 

### ISSUE THREAD

**Issue by owellandry** [muhammederdem/mini-player#10]
How could an API be implemented for the automatic playback of the player?

**Comment by biomassives** [comment]
yes it could. I am preparing this with supabase.io as backend graphql provider and will share.

**Comment by shailendramaurya** [comment]
You can have a look at this project.......
https://github.com/shailendramaurya/lovely
